pkg/sql/sem/tree/set.go

```diff
@@ -128,6 +151,11 @@ type SetTracing struct {
 	Values Exprs
 }
 
+// GetSetVar implements the SetClause interface.
+func (node *SetTracing) GetSetVar() (*SetVar, error) {
+	return nil, pgerror.Newf(pgcode.Syntax, "cannot set \"tracing\"  in this context")
```

Comment:
nit: extra space, you can avoid the escape by using backticks:

```
	return nil, pgerror.Newf(pgcode.Syntax, `cannot set "tracing" in this context`)

```

(also wonder whether

```

	return pgerror.Newf(pgcode.CantChangeRuntimeParam,
		"parameter %q cannot be changed", varName)
```

is better)